### ISSUE TITLE
[BUG] Only 2 Core Temperatures are shown if height is limited

### ISSUE THREAD

**Issue by moviuro** [aristocratos/btop#856]
**Describe the bug**

On btop 1.3.2 running in XTerm or kitty, if the height is below a certain threshold, core temperatures are only shown for cores 0 & 1.

**To Reproduce**

* Launch btop inside XTerm
* Set XTerm to 148 columns wide
* Set btop to preset 0
* Reduce height until temperature info disappears from the Cores

**Expected behavior**

Temperature value is always visible. (But temperature graphs should be hidden to save on space)

**Screenshots**

![xterm 2024 05 19 09 07 25](https://github.com/aristocratos/btop/assets/6882978/d7b24765-860a-421b-a542-98e685be2f2e)

*Above, the bug*

![xterm 2024 05 19 09 07 31](https://github.com/aristocratos/btop/assets/6882978/c512bde1-c2ca-4201-b966-cfa5ad37c156)

*The bug only triggers on limited height*

**Info (please complete the following information):**
 - btop++ version: `btop --version`: `btop version: 1.3.2`
   - If using snap: `snap info btop`
 - Binary: compiled by ALHP for archlinux (binary package built from source by a trusted third-party packager)
 - Architecture: [x86_64, aarch64, etc.] `uname -m`: `x86_64`
 - Platform: [Linux, FreeBSD, OsX] Linux
 - (Linux) Kernel: `uname -r`: `6.6.23-1.1-lts`
 - (OSX/FreeBSD) Os release version:
 - Terminal used: Xterm, kitty
 - Font used: [Go Mono](https://fontlibrary.org/en/font/go-mono)

**Additional context**

Contents of `~/.config/btop/btop.log`

```
2024/05/19 (09:26:12) | ===> btop++ v.1.3.2
2024/05/19 (09:26:12) | DEBUG: Running in DEBUG mode!
2024/05/19 (09:26:12) | INFO: Logger set to DEBUG
2024/05/19 (09:26:12) | DEBUG: Setting LC_ALL=en_US.UTF-8
2024/05/19 (09:26:12) | INFO: Running on /dev/pts/0
2024/05/19 (09:26:12) | INFO: Failed to load libnvidia-ml.so, NVIDIA GPUs will not be detected: libnvidia-ml.so.1: cannot open shared object file: No such file or directory
2024/05/19 (09:26:12) | WARNING: ROCm SMI: Failed to get maximum GPU temperature, defaulting to 110°C
2024/05/19 (09:26:12) | DEBUG: Shared::init() : Initialized.
2024/05/19 (09:26:18) | INFO: Quitting! Runtime: 00:00:06
```

**GDB Backtrace**

~~If btop++ is crashing at start the following steps could be helpful:~~

(Extra helpful if compiled with `make OPTFLAGS="-O0 -g"`)

1. run (linux): `gdb btop` (macos): `lldb btop`

2. `r` to run, wait for crash and press enter if prompted, CTRL+L to clear screen if needed.

3. (gdb): `thread apply all bt` (lldb): `bt all` to get backtrace for all threads

4. Copy and paste the backtrace here:

**Comment by LGTrader** [comment]
I am seeing this problem also. However I'm not sure it's height based as much as width based, but none the less I only see cpu 0 & 1 temps with my normal sized terminal on Kubuntu with is screen height. Expanding the width of the terminal to 149 shows all 32 temps, while at 148 or lower I only get the first 2.

**Comment by TopCheddar27** [comment]
Also seeing this problem

![image](https://github.com/aristocratos/btop/assets/73973868/5a68b304-a49e-449e-b058-382567ba3533)

**Comment by aristocratos** [comment]
@moviuro 
Should be fixed as of commit [3f384c0](https://github.com/aristocratos/btop/commit/3f384c0a5db78e4f622fdd7c4355a52c645f8eb2)

**Comment by moviuro** [comment]
Hi @aristocratos , I confirm that the latest git version fixes my issue! (516979b as of writing, built from https://aur.archlinux.org/packages/btop-git)
